QGCCommon.pri

```diff
@@ -232,13 +231,6 @@ CONFIG(debug, debug|release) {
 
 SOURCE_DIR = $$IN_PWD
 
-!iOSBuild {
```

Comment:
Also why is this removed?

Reply:
See this comment: https://github.com/mavlink/qgroundcontrol/pull/9136#issuecomment-715662938 - these assignments are not necessary with either QT version (`qmake` does all automatically) and break multi-ABI build in Qt 5.15, see: [QTBUG-81477](https://bugreports.qt.io/browse/QTBUG-81477). I've spent 1 week searching and fixing all of these. :wink:

Reply:
If you don't have this what is the directory structure?

Reply:
You also talk about multi-abi build. Are you talking about creating a package bundle which includes both 32 and 64 bit support? QGC doesn't support package bundles.  I'm not sure switching to it would be a positive thing. CI won't work with it either.

Reply:
> If you don't have this what is the directory structure you end up with and does that directory structure screw up the final installer step and/or the CI scripts which push things to the store. That code reaches into various places in the build treee.

As I said `qmake` crates exact same directory structure automatically with either 5.12 or 5.15 so works fine.

> You also talk about multi-abi build. Are you talking about creating a package bundle which includes both 32 and 64 bit support? 

It is possible but not necessary. Currently it works as is with 5.12. With 5.15 I'm adding `ANDROID_ABIS="arm64-v8a"` as an argument to `qmake` and it builds only 64 bit abi bundle as before.

>QGC doesn't support package bundles. CI won't work with it either.

You do not need to change anything in CI or support package bundles. Just add `ANDROID_ABIS="arm64-v8a"` and Qt does all automatically - apk produced still.